src/nginx/BUILD

```diff
@@ -121,5 +121,7 @@ cc_library(
         "//src/grpc:zero_copy_stream",
         "@nginx//:core",
         "@nginx//:http",
+        "@ngx_brotli//:http_brotli_filter",
+        "@ngx_brotli//:http_brotli_static",
```

Comment:
do we need static?  Here is what the doc say

ngx_brotli is a set of two nginx modules:

ngx_brotli filter module - used to compress responses on-the-fly,
ngx_brotli static module - used to serve pre-compressed files.

Do we need o support serve static compressed files.  ESP is just a proxy.

Reply:
I think I can see your point, but that I was assuming the functionality here is similar to [gzip](http://nginx.org/en/docs/http/ngx_http_gzip_module.html) and [gzip_static](http://nginx.org/en/docs/http/ngx_http_gzip_static_module.html). The latter which appears to be enabled in the nginx_esp binary.. (--with-http_gzip_static_module)